chasm/lib/callback/proto/v1/message.proto

```diff
@@ -33,6 +33,11 @@ message CallbackState {
 
     // Request ID that added the callback.
     string request_id = 9;
+
+    string workflow_id = 10;
+    string run_id = 11;
+
+    string namespace_id = 12;
```

Comment:
What is this for?

Reply:
Namespace is used to get timeout from dynamic config on a per-namespace basis

Reply:
Callback is not guaranteed to be attached to a workflow.
Wait for https://github.com/temporalio/temporal/pull/8533 to be merged and use `ExecutionKey()` instead.